model_compression_toolkit/data_generation/pytorch/optimization_functions/bn_layer_weighting_functions.py

```diff
@@ -18,30 +18,40 @@
 
 from model_compression_toolkit.data_generation.common.enums import BNLayerWeightingType
 from model_compression_toolkit.data_generation.pytorch.model_info_exctractors import OriginalBNStatsHolder, \
-    ActivationExtractor
+    ActivationExtractor, PytorchActivationExtractor
```

Comment:
Are you using ActivationExtractor here?

Reply:
In theory, you can. That is what we did in the research. Here we only have one function.